doc/releases/changelog-dev.md

```diff
@@ -31,38 +117,11 @@
   [SPSA gradient transform documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.gradients.spsa_grad.html) for details.
   Note: The full SPSA optimization method is already available as `SPSAOptimizer`.
 
-* Add the controlled CZ gate: CCZ.
+<h4>Custom measurement processes</h4>
 
-  ```pycon
-  >>> ccz = qml.CCZ(wires=[0, 1, 2])
-  >>> matrix = ccz.compute_matrix()
-  [[ 1  0  0  0  0  0  0  0]
-   [ 0  1  0  0  0  0  0  0]
-   [ 0  0  1  0  0  0  0  0]
-   [ 0  0  0  1  0  0  0  0]
-   [ 0  0  0  0  1  0  0  0]
-   [ 0  0  0  0  0  1  0  0]
-   [ 0  0  0  0  0  0  1  0]
-   [ 0  0  0  0  0  0  0 -1]]
-  ```
-
-  [(#3408)](https://github.com/PennyLaneAI/pennylane/pull/3408)
-
-* Add the controlled Hadamard gate.
-
-  ```pycon
-  >>> ch = qml.CH(wires=[0, 1])
-  >>> matrix = ch.compute_matrix()
-  [[ 1.          0.          0.          0.        ]
-   [ 0.          1.          0.          0.        ]
-   [ 0.          0.          0.70710678  0.70710678]
-   [ 0.          0.          0.70710678 -0.70710678]]
-  ```
-
-  [(#3408)](https://github.com/PennyLaneAI/pennylane/pull/3408)
-
-* Support custom measurement processes:
-  * `SampleMeasurement`, `StateMeasurement` and `MeasurementTransform` classes have been added.
+* Support custom measurement processes: 
+  TODO: need to expand on what these features mean for users
```

Comment:
Please remove once addressed

Reply:
These features could also benefit from some code examples (~2 total?)

Reply:
Addressed, No code examples because they are quite lengthy. Truth be told this feature benefits advanced users who want to add their own custom functionality into PL. It makes it easier for them to introduce new measurement quantities and have them be easily integrated with the rest of PennyLane